src/lib/homestore_backend/resync_pg_shard.fbs

```diff
@@ -0,0 +1,29 @@
+namespace homeobject;
+
+table Member {
+   uuid : [ubyte];
+   name : [ubyte];
+}
+
+table PGInfoEntry {
+    pg_id : uint32;                     // only low 16 bit is used for pg_id;
+    priority : int32;                   // pg priority;
+    replica_set_uuid : [ubyte];         // uuid of replica set
+    members : [Member];                 // peers;
+}
+
+table ShardInfoEntry {
+    state : ubyte;                      // shard state;
+    pg_id : uint32;                     // pg id which this shard belongs to;
+    shard_id : uint64;                  // shard id to be created with;
+    shard_size : uint64;                // shard size;
+    created_time : uint64;              // shard creation time
+    last_modified_time : uint64;        // shard last modify time
```

Comment:
as we agree to add created_lsn on shard, do we want to sync this ? 
or actually setting the created_lsn to snapshot_lsn is also right,  just sort of `inconsistency` we need to aware when we doing the scrubber.

Reply:
there should be no harm adding that, let me add it.